examples/cpp/Makefile

```diff
@@ -45,6 +48,18 @@ elliptic1D: elliptic1D.cpp
 
 elliptic2D: elliptic2D.cpp
 	$(CXX) $(CXXFLAGS) $(INCPATH) -o elliptic2D elliptic2D.cpp $(LIBS)
+	
+parabolic1D: parabolic1D.cpp
+	$(CXX) $(CXXFLAGS) $(INCPATH) -o parabolic1D parabolic1D.cpp $(LIBS)
+
+convection_diffusion: convection_diffusion.cpp
+	$(CXX) $(CXXFLAGS) $(INCPATH) -o convection_diffusion convection_diffusion.cpp $(LIBS)
+
+RK2: RK2.cpp
+	$(CXX) $(CXXFLAGS) $(INCPATH) -o RK2 RK2.cpp $(LIBS)
+	
+Schrodinger2D: Schrodinger2D.cpp	
+	$(CXX) $(CXXFLAGS) $(INCPATH) -o Schrodinger2D Schrodinger2D.cpp $(LIBS)
 
 clean:
-	rm -f transport1D schrodinger1D elliptic1D elliptic2D
+	rm -f transport1D schrodinger1D parabolic1D elliptic1D elliptic2D convection_diffusion RK2 Schrodinger2D
```

Comment:
Hi @arrowguy234 , thank you for updating the Makefile. However, it appears like your PR would re-introduce some changes that have already been introduced on `master` (specifically, the parabolic1D, convection_diffusion, and PK2 examples have already been merged, so those should not be highlighted in green). 

I recommend checking this [comment](https://github.com/csrc-sdsu/mole/pull/32#discussion_r1876858374) and the [following one](https://github.com/csrc-sdsu/mole/pull/32#discussion_r1885993495) that I left under one of your mates' PRs for reference on how to merge/rebase to incorporate `master` changes into your own branch. Please let me know if you have any questions.

Reply:
Sorry for some additional commits but i have fixed everything and made my fork up to date